faq/elastic-metal.mdx

```diff
@@ -1,10 +1,10 @@
 ---
 meta:
   title: Elastic Metal servers
-  description: Let us address the most common questions you have about Elastic Metal servers.
+  description: Discover Scaleway Elastic Metal servers, the first 100% cloud bare metal servers.
```

Comment:
Is it normal that we have bare metal written here? I guess it's for SEO?

Reply:
Yes, Vilena suggested this one